packages/itwinui-react/src/core/ColorPicker/ColorBuilder.tsx

```diff
@@ -305,6 +310,11 @@ export const ColorBuilder = React.forwardRef((props, ref) => {
 
       {showAlpha && (
         <Slider
+          thumbProps={() => {
+            return {
+              'aria-label': `Opacity slider`,
```

Comment:
You can remove the "slider" from the `aria-label`

```suggestion
              'aria-label': `Opacity`,
```